portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -1730,6 +1999,10 @@ export default function Environments() {
                                     {internalGateways && internalGateways.map((row) => (
                                         <Grid item xs={4}>
                                             <Card
+                                                style={{ opacity: allEnvRevision.some(o1 => {
+                                                    return o1.deploymentInfo.some(o2 => 
+                                                        o2.name === row.name && 
+                                                        o2.status === 'CREATED');})? '0.6' : '1' }}
```

Comment:
Let's fix the formatting here too for readability.

Reply:
fixed with https://github.com/wso2/apim-apps/pull/509/commits/feb7a470cc8167eb79976df580ae001188e69a10